src/tarantool-gdb.py

```diff
@@ -0,0 +1,468 @@
+        ( 'create_fk_constraint_parse_def::fkeys', 'fk_constraint_parse::link' ),
+        ( 'fiber::on_stop', 'trigger::link' ),
+        ( 'fiber::on_yield', 'trigger::link' ),
+        ( 'fiber::wake', 'fiber::state' ),
+        ( 'fiber_channel::waiters', 'fiber::state' ),
+        ( 'fiber_cond::waiters', 'fiber::state' ),
+        ( 'fiber_pool::idle', 'fiber::state' ),
+        ( 'func::func_cache_pin_list', 'func_cache_holder::link' ),
+        ( 'gc_checkpoint::refs', 'gc_checkpoint_ref::in_refs' ),
+        ( 'gc_state::checkpoints', 'gc_checkpoint::in_checkpoints' ),
+        ( 'index::full_scans', 'full_scan_item::in_full_scans' ),
+        ( 'index::nearby_gaps', 'gap_item::in_nearby_gaps' ),
+        ( 'iproto_thread::stopped_connections', 'iproto_connection::in_stop_list' ),
+        ( 'journal_queue::waiters', 'fiber::state' ),
+        ( 'latch::queue', 'fiber::state' ),
+        ( 'luaL_serializer::on_update', 'trigger::link' ),
+        ( 'mempool::cold_slabs', 'mslab::next_in_cold' ),
+        ( 'memtx_join_ctx::entries', 'memtx_join_entry::in_ctx' ),
+        ( 'memtx_story::reader_list', 'tx_read_tracker::in_reader_list' ),
+        ( 'memtx_story_link::nearby_gaps', 'gap_item::in_nearby_gaps' ),
+        ( 'point_hole_item::ring', 'point_hole_item::ring' ),
+        ( 'raft::on_update', 'trigger::link' ),
+        ( 'RebuildCkConstraints::ck_constraint', 'ck_constraint::link' ),
+        ( 'recovery::on_close_log', 'trigger::link' ),
+        ( 'replicaset::anon', 'replica::in_anon' ),
+        ( 'replicaset::on_ack', 'trigger::link' ),
+        ( 'replicaset::applier::on_rollback', 'trigger::link' ),
+        ( 'replicaset::applier::on_wal_write', 'trigger::link' ),
+        ( 'schema_module::funcs', 'func_c::item' ),
+        ( 'space::before_replace', 'trigger::link' ),
+        ( 'space::child_fk_constraint', 'fk_constraint::in_child_space' ),
+        ( 'space::ck_constraint', 'ck_constraint::link' ),
+        ( 'space::memtx_stories', 'memtx_story::in_space_stories' ),
+        ( 'space::on_replace', 'trigger::link' ),
+        ( 'space::parent_fk_constraint', 'fk_constraint::in_parent_space' ),
+        ( 'space::space_cache_pin_list', 'space_cache_holder::link' ),
+        ( 'swim::dissemination_queue', 'swim_member::in_dissemination_queue' ),
+        ( 'swim::on_member_event', 'trigger::link' ),
+        ( 'swim::round_queue', 'swim_member::in_round_queue' ),
+        ( 'swim_scheduler::queue_output', 'swim_task::in_queue_output' ),
+        ( 'tx_manager::read_view_txs', 'txn::in_read_view_txs' ),
+        ( 'tx_manager::all_stories', 'memtx_story::in_all_stories' ),
+        ( 'tx_manager::traverse_all_stories', 'memtx_story::in_all_stories' ), # struct rlist*
+        ( 'tx_manager::all_txs', 'txn::in_all_txs' ),
+        ( 'txn::conflict_list', 'tx_conflict_tracker::in_conflict_list' ),
+        ( 'txn::conflicted_by_list', 'tx_conflict_tracker::in_conflicted_by_list' ),
```

Comment:
just a little bit :)
I consider 80 symbols limit only as recommendation, not as a strict rule. Here we have a lot of items, one per line, so I assume it's not a big deal if a few of them cross 80 sym boundary. I think visually it's more consistent than it would be if some items occupy 2 line while others occupy 1.

I would keep current approach